test/jasmine/tests/range_slider_test.js

```diff
@@ -266,7 +266,7 @@ describe('Visible rangesliders', function() {
                 rangeslider: {visible: true}
             },
             width: 800,
-            hieght: 500
+            height: 500
```

Comment:
Good catch.
Let's remove this line, so that the default `height` is applied as before.